### PR TITLE
Add more options for client configuration customization

### PR DESCRIPTION
**Pull request by danielkza**

### CODE REVIEW

File: templates/client.json.j2 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "client": {
-    "name": "{{ ansible_hostname }}",
+    "name": "{{ sensu_client_name }}",
     "address": "{{ ansible_default_ipv4['address'] }}",
-      "subscriptions": {{ group_names | to_nice_json }}
+    "subscriptions": {{ sensu_client_subscriptions | to_nice_json(indent=6) }}
```
Comment (smuth4) [inline templates/client.json.j2:5]: Why does the indentation need to change?

Comment (danielkza) [reply templates/client.json.j2:5]: It was misaligned before. It's not strictly necessary, but I ended up fixing while testing it out. Feel free to revert it if you prefer :slightly_smiling_face:

Comment (smuth4) [reply templates/client.json.j2:5]: Ah I see what you mean now. I definitely agree it looks much nicer with the indentation, but I'm not ready to force people to use 2.2.X.

**Comment by smuth4** [comment]
Thanks for the PRs!

**Pull request merged**